### PR TITLE
Puerto Rico Flag

### PR DESCRIPTION
**Pull request by aallfredo**
Contribution a bit from me and a lot from @jorluiseptor

### CODE REVIEW

File: Flags/PuertoRico.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                line-height: 1%;
+                color: white;
+                left: 60px;
+                top: 50%;
+            }
+    </style>
+</head>
+<body>
+    <div id="pr"></div>
+
```
Comment (ManrajGrover) [inline Flags/PuertoRico.html]: Nit: Please remove extra lines

File: Flags/PuertoRico.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<html>
+<head>
+    <title>Puerto Rico | Flags</title>
+    <meta charset="utf-8" />
+    <style>
+        body {
+            padding: 20px;
+        }
+
+        #pr {
```
Comment (ManrajGrover) [inline Flags/PuertoRico.html]: Please change `id` to full name. :blush:

File: Flags/PuertoRico.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            width: 1235px;
+            height: 650px;
+            background: repeating-linear-gradient(180deg, #ce1126, #ce1126 150px, white 80px, white 250px);
+            box-shadow: 10px 10px 15px 0px #bbbbbb;
+            position: relative;
+            overflow: hidden;   
+        }
+        
+        #pr::before {         
+                position: absolute;
```
Comment (ManrajGrover) [inline Flags/PuertoRico.html]: Nit: Please follow 1 indentation.

**Comment by ManrajGrover** [comment]
@aallfredo Hey! This looks great! :smile: Thank you for contributing! I have left a few comments. Please fix them. Apart from that, please change `puertorico-flag.png` to `PuertoRico.png` and make same change in README.md.

Also, please resolve conflicts so that it can be merged. :smile:

**Comment by jorluiseptor** [comment]
I made these changes and submitted a pull request.

**Comment by ManrajGrover** [comment]
Closing this PR for #87

**Pull request closed without merging**